### PR TITLE
zh-translation:/docs/setup/platform-setup/gke/index.md

### PR DESCRIPTION
**Pull request by arunfung**
ref: https://github.com/servicemesher/istio-official-translation/issues/1295

### CODE REVIEW

**Comment by arunfung** [comment]
@googlebot I signed it!

File: content/zh/docs/setup/platform-setup/gke/index.md [hunk 1]
```diff
@@ -34,30 +34,26 @@ To enable SDS in Istio, use Kubernetes 1.13 or above.
     {{< /text >}}
 
     {{< tip >}}
-    The default installation of Mixer requires nodes with >1 vCPU. If you are
-    installing with the
-    [demo configuration profile](/docs/setup/additional-setup/config-profiles/),
-    you can remove the `--machine-type` argument to use the smaller `n1-standard-1` machine size instead.
+    默认安装 Mixer 要求节点的 vCPU 大于1。
```
Comment (rootsongjc) [inline content/zh/docs/setup/platform-setup/gke/index.md]: 数字与文字之间要有空格。

Comment (arunfung) [reply content/zh/docs/setup/platform-setup/gke/index.md]: 已修改

**Pull request merged**